src/explore-education-statistics-common/src/utils/array/intersperse.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Intersperse a separator item between each item in an array.
+ */
+export default function intersperse<TItem, TSeparator>(
```

Comment:
Not sure I'd agree there. Dictionary.com defines it as:

> to scatter here and there or place at intervals among other things:
_to intersperse flowers among shrubs._
>
> to diversify with something placed or scattered at intervals:
_to intersperse a dull speech with interesting anecdotes._

Merriam-Webster's is:

> 1: to insert at intervals among other things
_interspersing drawings throughout the text_
>
> 2: to place something at intervals in or among
_intersperse a book with pictures_

The definition is potentially up for debate here, but regardless, I mostly picked this name because this was the name of the util proposed for Lodash for this exact purpose: https://github.com/lodash/lodash/issues/2339

I'm not too keen on the proposed alternative so I'd prefer to stick with `intersperse` as it's nice, succinct and may be in Lodash one day.

Reply:
Fair enough, I read it more as 'to scatter here and there' rather than at regular intervals so it confused me a bit, but being consistent with a proposed lodash util makes sense.